### ISSUE TITLE
Color user interface needs a "reset selection" button

### ISSUE THREAD

**Issue by burnuser** [WordPress/gutenberg#49260]
### Description

The userinterface for color selection lacks a "reset selection" button. Which causes problems in usability and accessibility.

A) Using one of the predefined colors or gradients shows a checkmark in the selected color circle, but it is NOT obvious for all users (especially WordPress newbees or people with certain accessibility needs), that another click on the same color circle is necessary to reset the color selection.

B) Using a Custom color or gradient, there is not even the option to click on the same field again to reset the selection. As far as I know the only solution is clicking - first - on a predefined color/gradient and - second - clicking again on the same circle. (= Very strange UI!)

A simple "Reset selection" button could make all clear and easy to use.

### Step-by-step reproduction instructions

1. Add a simple paragraph block
2. Click in the Settings - Sidebar on Color - Background and select a predefined / custom color or gradient
3. Try to find the "Reset selection" option (as a newbee or a person with a screenreader in use)

### Screenshots, screen recording, code snippet

Unclear how to reset a predefined color selection:
![grafik](https://user-images.githubusercontent.com/32030147/226856051-00712465-40aa-4c0c-9756-27930652df65.png)

Even more unclear how to reset a custom color or a Gradient + strange and unnecessary complicated workflow, even if you know what to do:
![grafik](https://user-images.githubusercontent.com/32030147/226860489-5e41a94f-efd7-4291-bae8-025b607516ec.png)

### Environment info

WP 6.1.1 (with and without Gutenberg 15.3.1) + WP 6.2 RC 3

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes

**Comment by hanneslsm** [comment]
I agree. The usability would be easier if the button for removing the color or gradient would always be at the same place (instead of searching for the selected color and uncheck it).
Adobe has a great and easy to understand solution: White background and red diagonale line. This way we could bypass the discussion if "Reset" is really the best wording.
<img width="256" alt="image" src="https://user-images.githubusercontent.com/17861443/226865026-587db056-8506-4e4a-857c-e2f588120aa6.png">

**Comment by burnuser** [comment]
In contrast: The "Highlight" color userinterface for inline formatting in the block toolbar already has a dedicated "Clear" button.

To be consistent, the color userinterface in the sidebar could and should use the same feature and wording.

**Comment by Mamaduka** [comment]
Adding "Reset" is similar to the "Clear" option requested in your other issue, #49262.

The colors panel allows resetting a single or all colors using the "Options" ellipse menu in the title. Here's a previous discussion on removing separate "Clear" action - #42689.

@javierarce, @richtabor, is this something we're considering adding back to the UI?

**Comment by burnuser** [comment]
A) Reset/Clear is NOT similar to "Cancel" in #49262 because Rest/Clear means reverting a saved state. "Cancel" means don't save selection/customisation and leave all settings unchanged.

B) WOW! I'm using WordPress for many years, but until your hint I didn't even know the reset option in the ellipse menu. Good to know now for me, but it does not change anything on my issue. Not all users are aware how to reset colors.
+ The ellipse option resets ALL colors (Text + Background) and is useless if you want to rest only on of them.
So, please, add Reset/Clear option as a dedicated button like in the block toolbar.

**Comment by Mamaduka** [comment]
You can also reset only one color by clicking the name. It also says reset alongside the item name.

## Screenshot
![CleanShot 2023-03-22 at 19 40 53](https://user-images.githubusercontent.com/240569/226958731-33ba46f9-ce4b-4c19-a3ea-db6f03d036d2.png)

**Comment by burnuser** [comment]
Great to learn something new!
But I was unable to discover this by myself. And I think I'm not the only one with this special blind spot ...

**Comment by richtabor** [comment]
> @javierarce, @richtabor, is this something we're considering adding back to the UI?

No, I don't think so — but I do have an issue proposing to surface the reset action higher up: https://github.com/WordPress/gutenberg/issues/42196

**Comment by richtabor** [comment]
I'm in favor of closing this and https://github.com/WordPress/gutenberg/issues/49262 in favor of continuing the discussion on my https://github.com/WordPress/gutenberg/issues/42196 proposal.
